presto/dataset.py

```diff
@@ -197,6 +197,23 @@ def split_df(
         return train, val
 
 
+class WorldCerealDataset(WorldCerealBase):
+    # The simplest pd.DataFrame based dataset just
+    # for inference without labels
+    def __getitem__(self, idx):
```

Comment:
nit - could this just replace the currently not implemented `__getitem__` function in `WorldCerealBase` to limit how many extensions of the base class there are?

Reply:
That's a great suggestion! At some point all these datasets could anyway use a cleanup. But for now, moved `__getitem__` as suggested.